core/src/main/java/com/linecorp/armeria/server/docs/StructInfo.java

```diff
@@ -40,31 +41,64 @@
 public final class StructInfo implements NamedTypeInfo {
 
     private final String name;
-    private final List<FieldInfo> fields;
     @Nullable
+    private final String alias;
+    private final List<FieldInfo> fields;
     private final DescriptionInfo descriptionInfo;
 
     /**
      * Creates a new instance.
      */
     public StructInfo(String name, Iterable<FieldInfo> fields) {
-        this(name, fields, null);
+        this(name, null, fields, DescriptionInfo.empty());
     }
 
     /**
      * Creates a new instance.
      */
-    public StructInfo(String name, Iterable<FieldInfo> fields, @Nullable DescriptionInfo descriptionInfo) {
+    public StructInfo(String name, Iterable<FieldInfo> fields, DescriptionInfo descriptionInfo) {
+        this(name, null, fields, descriptionInfo);
+    }
+
+    /**
+     * Creates a new instance.
+     */
+    public StructInfo(String name, @Nullable String alias, Iterable<FieldInfo> fields,
+                      DescriptionInfo descriptionInfo) {
         this.name = requireNonNull(name, "name");
+        this.alias = alias;
         this.fields = ImmutableList.copyOf(requireNonNull(fields, "fields"));
-        this.descriptionInfo = descriptionInfo;
+        this.descriptionInfo = requireNonNull(descriptionInfo, "descriptionInfo");
     }
 
     @Override
     public String name() {
         return name;
     }
 
+    /**
+     * Returns the alias of the {@link #name()}.
+     */
+    @Nullable
+    @JsonInclude(Include.NON_NULL)
+    @JsonProperty
+    public String alias() {
```

Comment:
Can it be anything other than class name?

Reply:
Not currently.

Reply:
If so, how about renaming it to something like `fqcn` so people do not wonder what it means?

Reply:
Actually, `name()` is also a fully qualified name.
`alias()` is used for the generated classes by protobuf compiler. 
The protobuf's FQCN is set to `name()` and the generated class name is set to `alias()`
to link both names in `DocService`.
https://github.com/line/armeria/blob/11ee1ff1c5e830a3c52e1e09e452618357fbe2dd/protobuf/src/main/java/com/linecorp/armeria/server/protobuf/ProtobufNamedTypeInfoProvider.java#L108
https://github.com/line/armeria/blob/11ee1ff1c5e830a3c52e1e09e452618357fbe2dd/scalapb/scalapb_2.13/src/main/scala/com/linecorp/armeria/server/scalapb/ScalaPbNamedTypeInfoProvider.scala#L33
If `alias()` looks weird, I was also considered `nickName()`. 😅

Reply:
Sure.